### ISSUE TITLE
killer is not defined

### ISSUE THREAD

**Issue by israelglar** [gilamran/tsc-watch#53]
I'm having this error:

```js
  ReferenceError: killer is not defined
  at kill (node_modules/tsc-watch/lib/runner.js:20:20)
  at killProcesses (node_modules/tsc-watch/lib/tsc-watch.js:24:21)
  at Socket.tscProcess.stdout.on.buffer (node_modules/tsc-watch/lib/tsc-watch.js:51:5)
  at Socket.emit (events.js:189:13)
  at addChunk (_stream_readable.js:284:12)
  at readableAddChunk (_stream_readable.js:265:11)
  at Socket.Readable.push (_stream_readable.js:220:10)
  at Pipe.onStreamRead [as onread] (internal/stream_base_commons.js:94:17)
node_modules/tsc-watch/lib/runner.js:20
  return Promise.all([killer(process), exitPromise]);
```

The file:  `tsc-watch/lib/runner.js`
is missing this line, perhaps?
```js
const killer = require('./killer')
```

**Comment by gilamran** [comment]
Please verify with the latest version.

**Comment by pronebird** [comment]
I think 2.1.0 has this bug, see:

https://github.com/gilamran/tsc-watch/blob/master/lib/runner.js#L20

**Comment by israelglar** [comment]
Yes, this is with the latest version, 2.1.0

**Comment by gilamran** [comment]
Hoo my bad!
Fixed on version 2.1.1
